src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRequestPrebidCacheVastxml.java

```diff
@@ -8,4 +8,6 @@
 public class ExtRequestPrebidCacheVastxml {
 
     Integer ttlseconds;
+
+    Boolean returncreative;
```

Comment:
Check letter case: `returnCreative`

Reply:
I followed `ttlseconds` style, maybe change it too?

Reply:
The name `returnCreative` is defined in documentation.
Leave `ttlseconds` as is.